espresso-support/demo/src/androidTest/java/com/novoda/movies/MovieItemViewTalkBackTest.java

```diff
@@ -57,6 +61,18 @@ public void clickMovieItemView() {
         checkMenuDisplayed();
     }
 
+    @Test
+    public void movieItemViewHasCustomUsageHintOnClick() {
+        onView(withClassName(is(MovieItemView.class.getName())))
+                .check(matches(withUsageHintOnClick("see actions")));
+    }
+
+    @Test
+    public void movieItemViewHasCustomUsageHintOnLongClick() {
+        onView(withClassName(is(MovieItemView.class.getName())))
+                .check(matches(withUsageHintOnLongClick("open details")));
```

Comment:
Given they're asserting against stuff that is meant to be read by TTS that out rather than read by users, is case sensitivity necessary?